packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -33,12 +38,11 @@ const editorSettings = {
   useSoftTabs: true,
   minLines: 1,
   maxLines: 10,
-  fontSize: 12,
```

Comment:
Not sure why it was 12 before. It doesn't match other leafygreen inputs in the same toolbar, they all have default 13px font size

Reply:
Ah nice catch!